lib/shared/utils/image_strings.dart

```diff
@@ -0,0 +1,5 @@
+class FlexImages {
+  // TODO Replace with the media permalink corresponding to this image
```

Comment:
We can find it in our backoffice now, right?

Reply:
Not sure yet, our backoffice is half down. Can wait for a future PR.